adafruit_motor/servo.py

```diff
@@ -83,19 +96,19 @@ class Servo(_BaseServo):
 """
     def __init__(self, pwm_out, *, actuation_range=180, min_pulse=750, max_pulse=2250):
         super().__init__(pwm_out, min_pulse=min_pulse, max_pulse=max_pulse)
-        self._actuation_range = actuation_range
+        self.actuation_range = actuation_range
```

Comment:
Add a triple quoted doc string here so it is included in RTD.

Reply:
Since the `Servo` class has a docstring, it's already in RTD. See https://circuitpython.readthedocs.io/projects/motor/en/latest/api.html#adafruit_motor.servo.Servo.

Alphabetically `ContinuousServo` came first -- maybe that was confusing?

Reply:
It does for the argument but not the object attribute (like `angle`).

Reply:
I'm confused. Here is the `angle` doc: https://circuitpython.readthedocs.io/projects/motor/en/latest/api.html#adafruit_motor.servo.Servo.angle